integrationTests/chainSimulator/common.go

```diff
@@ -289,7 +288,7 @@ func IssueFungible(
 	txResult := SendTransaction(t, cs, sender, nonce, vm.ESDTSCAddress, issueCost, issueArgs, uint64(60000000))
 	RequireSuccessfulTransaction(t, txResult)
 
-	return getEsdtIdentifier(t, nodeHandler, tokenTicker, core.FungibleESDT)
+	return getEsdtIdentifier(t, cs.GetNodeHandler(core.MetachainShardId), tokenTicker, core.FungibleESDT)
```

Comment:
This will break sovereign integration tests if the function is used. Should be reverted.

Reply:
Correct. Reverted